pkg/mpc/cmd/automation.go

```diff
@@ -40,24 +44,54 @@ var automationRetrieveCmd = &cobra.Command{
 	},
 }
 
-func aliasNormalizeFunc(_ *pflag.FlagSet, name string) pflag.NormalizedName {
+// automationStatusCmd represents  status command
+var automationUpdateCmd = &cobra.Command{
+	Use:   "update",
+	Short: "Automation retrieve",
+
+	Run: func(cmd *cobra.Command, args []string) {
+		file, err := ioutil.ReadFile(file)
+		exitOnErr(err)
+		data := opsmanager.AutomationConfig{}
+
+		err2 := json.Unmarshal([]byte(file), &data)
```

Comment:
nit: maybe use `json.NewDecoder` like in other places

Reply:
They a bit different as readfile returns `byte[]` and the decoder expects an io.Reader